#make a code block with exception handling
try:
    foo